Common/Core/aodMerger.cxx

```diff
@@ -93,6 +99,16 @@ int main(int argc, char* argv[])
     printf("Processing input file: %s\n", line.Data());
 
     auto inputFile = TFile::Open(line);
+    if (!inputFile) {
+      printf("Could not open input file %s.\n", line.Data());
+      if (skipNonExistingFiles) {
+        continue;
+      } else {
+        printf("Aborting merge!\n");
```

Comment:
I think we should also close the output file and unlink it because it is only partial.

Reply:
by breaking out of the while loop all buffered trees are now saved and the file is closed properly